src/pages/components/controls/filenameControls.tsx

```diff
@@ -8,25 +10,61 @@ type PatternTokenProps = {
 }
 
 export const PatternToken = memo(
-  ({ tokenName, isOn, handleChange }: PatternTokenProps) => {
-    const [state, setState] = useBoolean(isOn)
+  ({ tokenName, isOn, handleChange }: PatternTokenProps) => (
+    <Box
+      pl={'1em'}
+      pr={'1em'}
+      fontSize={'1em'}
+      fontWeight={'medium'}
+      color={'gray.900'}
+      borderRadius={'full'}
+      bg={isOn ? 'token.active' : 'white'}
+      cursor="pointer"
+      onClick={() => handleChange(!isOn)}
+    >
+      {tokenName}
+    </Box>
+  )
+)
+
+type SortablePatternTokenProps = {
+  token: string
+  name: string
+  handleChange?: (state: boolean) => void
```

Comment:
Since the only usage of this function is removing the token from the slot,
it needs a more intuitive name and should be required property.
e.g.
```js
handleClose: () => void
handleRemove: () => void
```